integreat_cms/cms/templatetags/content_filters.py

```diff
@@ -100,7 +100,7 @@ def minor_edit_label(region: Region, language: Language) -> Promise:
     if language_node.is_leaf():
         return _("Implications on the translation status")
     if language_node.is_root():
-        return _("Implications on other translations")
+        return _("Implications on translations")
     return _("Implications on the status of this and other translations")
```

Comment:
@dkehne I assume that you also wanted to make the heading more clear for the languages other than the default language (e.g. English, Arabic etc.), right?

At the moment, this looks like this:

![Screenshot 2024-03-26 at 17-55-22 Integreat Redaktionssystem](https://github.com/digitalfabrik/integreat-cms/assets/16021269/0edf87c6-2eb6-4890-9afc-885d8325ed85)

I guess this should probably also be changed to "Implications on translations"?

Reply:
yep, thank you for your review. I will work on the rest :)